test.sh

```diff
@@ -1,7 +1,7 @@
 echo "record type"
 
 # term query
-# curl 'http://localhost:9200/test1/_search?pretty' -d '{
+# curl -X GET 'http://localhost:9200/test1/_search?pretty' -H 'Content-Type: application/json' -d '{
```

Comment:
This is the same thing I had to tweak in my ES script. This and I had to see `pretty=true` rather than `pretty=1`. Your code makes it look like the value isn't even necessary as long as the param is present though. Correct?

Reply:
That's how it used to work in ES5 and I don't think that I noticed any smashed output when I was working on this.

Looking through the ES 6 docs, it looks like they do specify "true" but they didn't used to need it for the earlier branches (https://www.elastic.co/guide/en/elasticsearch/guide/current/get-doc.html)  I can make that change very quickly and push.

Reply:
This is something I'll need to clean up in the data repository as well

Reply:
They show it without a value in the ES6 docs as well: https://www.elastic.co/guide/en/elasticsearch/reference/current/_index_and_query_a_document.html
:man_shrugging:

Reply:
Well shucks, I wish I hadn't changed it now.  This was the ES 6 documentation that I came across when I was trying to figure out what they preferred `pretty` to look like in this version and they specified true:  https://www.elastic.co/guide/en/elasticsearch/reference/6.4/common-options.html#_pretty_results

Reply:
Ah. Well that link is probably a better thing to reference than examples scattered elsewhere in the docs. They probably just haven't gone through to make examples match that page's recommendations.